README.md

```diff
@@ -41,6 +41,7 @@ Like `awesome-hyperterm`? Reach out to me and say *hi* on [Twitter](https://twit
 
 # Themes
 * [hyperterm-atom-dark](https://www.npmjs.com/package/hyperterm-atom-dark) - Dark - Really beautiful import of Atom One Dark theme from the [official Atom theme](https://github.com/atom/one-dark-syntax).
+* [hyperblue](http://npmjs.com/package/hyperblue) – Dark, cool hues. Turns HyperTerm blue.
```

Comment:
Can you change it to `https://npmjs.com` please? HTTP throws a redirect. Want to keep the entire list on the HTTPS for the sake of consistency.